chapters/annotations.tex

```diff
@@ -475,6 +477,18 @@ \section{Annotations for Simulation Experiments}\label{annotations-for-simulatio
 
 The \lstinline!experiment! annotation defines the default start time (\lstinline!StartTime!) in {[}s{]}, the default stop time (\lstinline!StopTime!) in {[}s{]}, the suitable time resolution for the result grid (\lstinline!Interval!) in {[}s{]}, and the default relative integration tolerance (\lstinline!Tolerance!) for simulation experiments to be carried out with the model or block at hand.  If \lstinline!StartTime! is not specified it is assumed to be \lstinline!0.0!.
 
+\subsection{Annotation for Test Cases}\label{annotation-for-test-cases}
+\begin{lstlisting}[language=grammar]
+testcase-annotation:
+   annotation "(" "TestCase" "(" "shouldPass" "=" ( false | true ) ")" ")"
+\end{lstlisting}
+If \lstinline!shouldPass! is \lstinline!false! it indicates that the translation or the simulation of the model should fail.
+If a tools checks a package where classes have \lstinline!shouldPass=false! they should not generate errors - and testing them may even be skipped; if a model with \lstinline!shouldPass=false! translates and simulates without error that should generate a diagnostics.
```

Comment:
I find the dash before _and_ a bit odd, how about a comma?  Perhaps also split into two sentences?

I also find the diagnostics part maybe a bit too challenging for tools to handle, and the information in the annotation insufficient to directly result in tests of much value.  (Separation into `shouldTranslate` and `shouldSimulate` would have resulted in somewhat better tests.)  Isn't it better to say more generally that the model may be useful for purposes of negative testing?

```suggestion
If a tools checks a package where classes have \lstinline!shouldPass=false! they should not generate errors, and may even be skipped.
On the other hand, models with \lstinline!shouldPass=false! may be useful for creation of negative tests in tool-specific ways.
```

Reply:
Good suggestion - but I will improve 'may even be skipped' to include that it's checking that is skipped.